src/Components/RecsListTable/RecsListTable.spec.ct.js

```diff
@@ -282,6 +282,45 @@ describe('successful non-empty recommendations list table', () => {
       .find('td[data-label=Category]')
       .should('contain', 'Service Availability');
   });
+
+  it('the Impacted filters work correctly', () => {
+    const RECS_LIST_TABLE = 'div[id=recs-list-table]';
```

Comment:
RECS_LIST_TABLE is already accessible and defined in the outer scope on line 12: https://github.com/RedHatInsights/ocp-advisor-frontend/blob/e8e096dfd4827ac52203391cda8bc899bdeb012b/src/Components/RecsListTable/RecsListTable.spec.ct.js#L12